pennylane_qiskit/converter.py

```diff
@@ -328,56 +327,62 @@ def _function(*args, params: dict = None, wires: list = None, **kwargs):
         wire_map = map_wires(qc_wires, wires)
 
         # Stores the measurements encountered in the circuit
-        mid_circ_meas, terminal_meas = [], []
+        terminal_meas = []
+        mid_circ_meas, mid_circ_regs = [], {}
```

Comment:
Added a comment for the explanation. It is basically a dictionary that maps the classical register used by Qiskit to the measurements we are doing so that we know which ones to use for conditioning the control flows.